src/client/src/App.js

```diff
@@ -22,23 +24,25 @@ class App extends Component {
         }
     }
     onKeyPress = event => {
-         if (event.key === ' ') {
-              if (this.state.text === ' ') {
-                   this.setState({text: ''});
-              } else {
-                   this.handleSubmit();
-              }
-         }
+         // if (event.key === ' ') {
+         //      if (this.state.text === ' ') {
+         //           this.setState({text: ''});
+         //      } else {
+         //           this.handleSubmit();
+         //      }
+         // }
+         this.handleSubmit();
     }
     handleSubmit = async e => {
         if (e) {
              e.preventDefault();
         }
-        let newMessage = new Message(this.state.user, this.state.text)
-        this.setState(previousState => ({
```

Comment:
Believe we still need a call to update the React state

Reply:
which state should we update in handleSubmit?

Reply:
`this.state.conversation` and `this.state.text`